### ISSUE TITLE
[Android] Text with markdown is broken in most of the components

### ISSUE THREAD

**Issue by Gunashree-R-V** [microsoft/AdaptiveCards#4463]
# Platform

* Android

# Author or host

Teams mobile

# Version of SDK

1.2.10

# Details

Sample [**Agenda**](https://adaptivecards.io/samples/Agenda.html) card, [**Expense report**](https://adaptivecards.io/samples/ExpenseReport.html) card available in public documentation is broken experience.

Here is the comparison of experience with 1.2.6(SDK version) and 1.2.10(SDK version).

| 1.2.6 | 1.2.10|
|-----------|:-----------:|
| ![image](https://user-images.githubusercontent.com/28745768/88173893-fb0b0c80-cc40-11ea-8003-2e1f4964869b.png) | ![image](https://user-images.githubusercontent.com/28745768/88174060-3b6a8a80-cc41-11ea-83d4-d8b37b4a5ff6.png) | 

`TextBlock`, `FactSet` with value enclosed with markdown support are completely broken.

Here is another example:
| 1.2.6 | 1.2.10|
|-----------|:-----------:|
| ![image](https://user-images.githubusercontent.com/28745768/88176728-56d79480-cc45-11ea-9b36-4919d8c985b0.png)| ![image](https://user-images.githubusercontent.com/28745768/88176614-2db70400-cc45-11ea-82cf-13b3ba72851b.png)| 
|  ![image](https://user-images.githubusercontent.com/28745768/88177236-2512fd80-cc46-11ea-93ff-43bc309ed8d5.png)  |![image](https://user-images.githubusercontent.com/28745768/88177265-30febf80-cc46-11ea-91cf-68b66efc1911.png)| 

Attaching the Card JSON for the same:

```
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "TextBlock",
            "text": "_Italic_"
        },
        {
            "type": "TextBlock",
            "text": "**Bold**"
        },
        {
            "type": "TextBlock",
            "text": "**~~Bold, strike-through text~~**"
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.2"
}
```

cc: @VikrantSingh01, @anbah34 ,@shbhadra

**Comment by shalinijoshi19** [comment]
@golddove can you help prioritize this one today please? Thanks!

**Comment by VikrantSingh01** [comment]
Team, Any update on the fix E.T.A for this blocker bug from v1.2.10 ?

**Comment by golddove** [comment]
@VikrantSingh01 we will include this fix in the v1.2.11 patch (scheduled end of month)
